app/assets/stylesheets/variables.scss

```diff
@@ -1,4 +1,6 @@
-@import './variables/colors';
-
+$blue-light: #9acbff;
+$gray-light: #b8b8b8;
+$gray-lighter: #ddd;
+$white-dark: #f0f0f0;
```

Comment:
I was running into errors when these variables were in a separate file, so I'm just inlining them here instead.